### PR TITLE
Update nz.m3u

### PR DESCRIPTION
**Pull request by Sphinxroot**
added 2 channel + some fixe

### CODE REVIEW

File: channels/nz.m3u [hunk 1]
```diff
@@ -1,27 +1,33 @@
...
+#EXTINF:-1 tvg-id="MaoriTelevision.au" tvg-name="Maori Television" tvg-country="NZ;AU" tvg-language="English" tvg-logo="https://freeviewnz.tv/nonumbracoimages/ChannelsOpg/freeview-channel-logos-38.png" group-title="",Maori Television
 https://bcsecurelivehls-i.akamaihd.net//hls/live/720612/1614493167001_1/master_iPad1080p.m3u8
-#EXTINF:-1 tvg-id="MaoriTV.nz" tvg-name="Maori TV" tvg-country="NZ" tvg-language="" tvg-logo="" group-title="",Maori TV
+#EXTINF:-1 tvg-id="MaoriTV.nz" tvg-name="Maori TV" tvg-country="NZ" tvg-language="Maori" tvg-logo="https://freeviewnz.tv/nonumbracoimages/ChannelsOpg/freeview-channel-logos-38.png" group-title="",Maori TV
 https://bcsecurelivehls-i.akamaihd.net/hls/live/720612/1614493167001_1/master_iPad720p.m3u8
 #EXTINF:-1 tvg-id="tv.13" tvg-name="Māori Television" tvg-country="NZ;AU" tvg-language="Maori" tvg-logo="https://www.freeviewnz.tv/nonumbracoimages/ChannelsOpg/freeview-channel-logos-38.png" group-title="",Māori Television (1080p)
 https://bcsecurelivehls-i.akamaihd.net//hls/live/720612/1614493167001_1/master.m3u8
```
Comment (LaneSh4d0w) [inline channels/nz.m3u:17]: this one is the master link. accepting request but it'll go after cleaning.

File: channels/nz.m3u [hunk 2]
```diff
@@ -1,27 +1,33 @@
...
+https://ptvlive.kordia.net.nz/out/v1/daf20b9a9ec5449dadd734e50ce52b74/index.m3u8
 #EXTINF:-1 tvg-id="Rialto.nz" tvg-name="Rialto" tvg-country="NZ;AU" tvg-language="English" tvg-logo="https://i.imgur.com/RnzWT9C.jpg" group-title="",Rialto (1080p)
 https://rialto-rialto-samsungaustralia.amagi.tv/playlist.m3u8
-#EXTINF:-1 tvg-id="TeReo.nz" tvg-name="Te Reo" tvg-country="NZ" tvg-language="" tvg-logo="http://www.freeviewnz.tv/nonumbracoimages/ChannelsOpg/te-reo.png" group-title="",Te Reo
+#EXTINF:-1 tvg-id="TeReo.nz" tvg-name="Te Reo" tvg-country="NZ" tvg-language="Maori" tvg-logo="http://www.freeviewnz.tv/nonumbracoimages/ChannelsOpg/te-reo.png" group-title="",Te Reo
 https://bcsecurelivehls-i.akamaihd.net//hls/live/720613/1614493167001_2/master_iPad1080p.m3u8
-#EXTINF:-1 tvg-id="TeReo.nz" tvg-name="Te Reo" tvg-country="NZ" tvg-language="" tvg-logo="" group-title="",Te Reo
+#EXTINF:-1 tvg-id="TeReo.nz" tvg-name="Te Reo" tvg-country="NZ" tvg-language="" tvg-logo="http://www.freeviewnz.tv/nonumbracoimages/ChannelsOpg/te-reo.png" group-title="",Te Reo
 https://bcsecurelivehls-i.akamaihd.net/hls/live/720613/1614493167001_2/master_iPad720p.m3u8
```
Comment (LaneSh4d0w) [inline channels/nz.m3u:29]: same case here, master link is  https://bcsecurelivehls-i.akamaihd.net/hls/live/720613/1614493167001_2/master.m3u8

File: channels/nz.m3u [hunk 3]
```diff
@@ -1,27 +1,33 @@
 #EXTM3U
+#EXTINF:-1 tvg-id="TheBreeze.nz" tvg-name="Breeze TV NZ" tvg-country="NZ" tvg-language="English" tvg-logo="https://i.imgur.com/3dpH9XF.jpg" group-title="Music",Breeze TV (720p)
```
Comment (LaneSh4d0w) [inline channels/nz.m3u:2]: Incoherency between tvg-id and tvg-name. Either one or the other name.

File: channels/nz.m3u [hunk 4]
```diff
@@ -1,27 +1,33 @@
...
+https://livestreamdirect-breezetv.mediaworks.nz/breezetv.m3u8
+#EXTINF:-1 tvg-id="TheEdge.nz" tvg-name="The Edge NZ" tvg-country="NZ" tvg-language="English"  tvg-logo="https://i.imgur.com/ZHJVvOj.png" group-title="Music",The Edge (720p)
```
Comment (LaneSh4d0w) [inline channels/nz.m3u:4]: No need to keep the "NZ" next to the name of the channel. The channel to itself is well located :)

**Comment by LaneSh4d0w** [comment]
I've fixed your playlist and removed dupe links.

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**